pkg/loader/load.go

```diff
@@ -418,9 +418,19 @@ func (s *loaderImpl) processMysqlDDL(ddl *DDL) error {
 			}
 		}
 
-		if _, err = tx.Exec(ddl.SQL); err != nil {
+		sql, err := removeDDLPlacementOptions(ddl.SQL)
+		if err != nil {
+			return err
+		}
```

Comment:
Can we just warn this error here and execute the origin SQL? just like we did in
https://github.com/pingcap/tidb-binlog/blob/194d4ac/pkg/util/util.go#L343

Reply:
fine

Reply:
done